docs/pages/API/Button/index.js

```diff
@@ -0,0 +1,296 @@
+import ButtonCustomizationsSpecimen from '../../../specimens/ButtonCustomizations'
+import ButtonModesSpecimen from '../../../specimens/ButtonModes'
+import ButtonSizesSpecimen from '../../../specimens/ButtonSizes'
+import ButtonStylesSpecimen from '../../../specimens/ButtonStyles'
+import style from '../style.less'
+
+const apiOptions = [{
+   name: 'borderRadius',
+   type: 'String',
+   default: '4px',
+   description: 'Set the border-radius of the button. By default, all buttons inherit their borderRadius from the <code>elementBorderRadius</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'className',
+   type: 'String',
+   default: '-',
+   description: 'Pass a <code>className</code> to customize className of the component',
+   responsive: false
+}, {
+   name: 'color',
+   type: 'String',
+   default: '-',
+   description: 'The primary button color (usually its background color). Pass any css compatible color value. By default, all buttons inherit their color from the <code>highlightColor</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'fontFamily',
+   type: 'String',
+   default: '-',
+   description: 'Apply a specific font-family to the text within this component. By default, all buttons inherit the <code>fontFamily</code> set on the parent <code>OIO</code> container',
+   responsive: true
+}, {
+   name: 'mode',
+   type: 'Enum',
+   default: '-',
+   description: 'Availalble options include: <code>disabled</code>, <code>normal</code>, <code>pulsing</code>, <code>loading</code>',
+   responsive: true
+}, {
+   name: 'name',
+   type: 'String',
+   default: '-',
+   description: 'Text that appears inside the button',
+   responsive: true
+}, {
+   name: 'onClick',
+   type: 'Function',
+   default: '-',
+   description: 'Function to execute on mouse click',
+   responsive: false
+}, {
+   name: 'outline',
+   type: 'Boolean',
+   default: '<code>false</code>',
+   description: 'Set to <code>true</code>, gives the button an outline appearance',
+   responsive: true
+}, {
+   name: 'padding',
+   type: 'String',
+   default: '-',
+   description: 'Set the horizontal padding of the button',
+   responsive: true
+}, {
+   name: 'rounded',
+   type: 'Boolean',
+   default: '<code>false</code>',
+   description: 'Set to <code>true</code>, gives the button a rounded pill-like appearance',
+   responsive: true
+}, {
+   name: 'size',
+   type: 'Enum',
+   default: '<code>3</code>',
+   description: 'Set the <code>Button</code> component size. Available values include:  <code>lg</code>, <code>md</code>, <code>sm</code>, <code>xs</code>',
+   responsive: true
+}, {
+   name: 'style',
+   type: 'Object',
+   default: '<code>{}</code>',
+   description: 'Pass a style <code>object</code> to customize inline style of the <code>Button</code> component',
+   responsive: true
+}, {
+   name: 'textColor',
```

Comment:
I noticed that this does not work when `outline` is used, and that `color` dictates the stroke and text color. What should happen here?